_data/sw_clients.yml

```diff
@@ -818,6 +818,39 @@
           sasl-3.2:
           server-time:
           userhost-in-names:
+    - name: soju (as Server)
+      link: https://sr.ht/~emersion/soju
+      support:
+        stable:
+          cap-3.1:
+          cap-3.2:
+          cap-notify:
+          draft/chathistory:
+          away-notify:
+          batch:
+          echo-message:
+          extended-join:
+          message-tags:
+          msgid:
+          multi-prefix:
+          sasl-3.1:
+          sasl-3.2:
+          server-time:
+    - name: soju (as Client)
+      link: https://sr.ht/~emersion/soju
+      support:
+        stable:
```

Comment:
Could add a `SASL` entry below, as we support both PLAIN and EXTERNAL. See the entry for znc which also support both for instance.

Reply:
Done!